apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.py

```diff
@@ -128,64 +177,24 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             "editable": editable,
             "info": info,
             "size": size,
-            "show_info": pl.get_boolean_attrib(
-                element, "show-help-text", SHOW_HELP_TEXT_DEFAULT
-            ),
+            "show_info": show_info,
             "uuid": pl.get_uuid(),
             display.value: True,
             "placeholder": placeholder,
             "raw_submitted_answer": raw_submitted_answer,
             "type": bigo_type,
+            "parse_error": parse_error,
+            "display_append_span": show_info or parse_error,
         }
 
-        if score is not None:
+        if show_score and score is not None:
             score_type, score_value = pl.determine_score_params(score)
             html_params[score_type] = score_value
 
         with open(BIG_O_INPUT_MUSTACHE_TEMPLATE_NAME, "r", encoding="utf-8") as f:
             return chevron.render(f, html_params).strip()
 
     elif data["panel"] == "submission":
-        parse_error: Optional[str] = data["format_errors"].get(name)
```

Comment:
Note: These lines preparing the parse error were moved above, outside of this if statement (since the parse error is used in multiple panels now).